src/components/Explore/Explore.tsx

```diff
@@ -80,14 +80,15 @@ function getLabelLength(series: Series, shortLabel: boolean) {
 }
 
 const Explore: React.FunctionComponent<{
-  fips: string;
-}> = ({ fips }) => {
+  fipsList: string[];
```

Comment:
maybe "initial fips list" for clarity?  i was having trouble figuring out whether this was controlling the module or was just the initial list

Reply:
Makes sense, will update 👍